packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -46,70 +55,56 @@ The [Brands API](/docs/reference/api/brands/) is a feature (currently in Early A
 
 - If you customize an email template, you need to manually create a translation for each additional language that you support in your org.
 
-## Add or edit email templates
+## Edit a default email template
 
 Use these steps to add or edit a template in one of the Okta-supported languages.
 
 > **Note:** Free trial editions of Okta can't create or send customized email templates.
 
-1. In the Admin Console, go to **Settings** and then **Emails & SMS**. The **Email** tab is selected by default.
-2. From the left pane, select the email template that you want to edit or for which you want to add a new translation. A text version of the default message appears.
-3. Click **Edit** to open the message in HTML.
-4. Make your edits, and then click **Save**. The default language version of your edited message appears in the **Custom Email** table.
-5. If your org supports additional languages, click **Add Translation**, and then select a language from the drop-down box.
-6. Make your edits, and then click **Add Translation**.
-7. Repeat steps 5 and 6 for additional languages.
-
-## Localization notes
+1. In the Admin Console, go to **Customizations** > **Emails**.
+1. Click a default email template listed in the left pane.
+1. Click **Edit** to open the message in HTML. If you see **Add Translation** instead of **Edit**, the template has already been customized. You need to make any additional edits in the default language version. Skip to Step 2 in [Add translations](#add-translations).
```

Comment:
```suggestion
1. Click **Edit** to open the message in HTML. If you see **Add Translation** instead of **Edit**, the template is already customized. You need to make any additional edits in the default language version. Skip to step 2 in [Add translations](#add-translations).
```